sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporterBuilder.java

```diff
@@ -6,79 +6,33 @@
-import com.azure.monitor.opentelemetry.exporter.implementation.localstorage.LocalStorageStats;
-import com.azure.monitor.opentelemetry.exporter.implementation.localstorage.LocalStorageTelemetryPipelineListener;
-import com.azure.monitor.opentelemetry.exporter.implementation.models.ContextTagKeys;
-import com.azure.monitor.opentelemetry.exporter.implementation.pipeline.TelemetryItemExporter;
-import com.azure.monitor.opentelemetry.exporter.implementation.pipeline.TelemetryPipeline;
-import com.azure.monitor.opentelemetry.exporter.implementation.pipeline.TelemetryPipelineListener;
-import com.azure.monitor.opentelemetry.exporter.implementation.utils.ResourceParser;
-import com.azure.monitor.opentelemetry.exporter.implementation.utils.TempDirs;
-import com.azure.monitor.opentelemetry.exporter.implementation.utils.VersionGenerator;
-import io.opentelemetry.sdk.autoconfigure.ResourceConfiguration;
-import io.opentelemetry.sdk.logs.export.LogRecordExporter;
-import io.opentelemetry.sdk.metrics.export.MetricExporter;
-import io.opentelemetry.sdk.resources.Resource;
-import io.opentelemetry.sdk.trace.export.SpanExporter;
+import io.opentelemetry.sdk.autoconfigure.AutoConfiguredOpenTelemetrySdkBuilder;
 
-import java.io.File;
 import java.util.ArrayList;
 import java.util.List;
-import java.util.Map;
 import java.util.Objects;
 
-import static java.util.concurrent.TimeUnit.MINUTES;
-
 /**
- * This class provides a fluent builder API to instantiate {@link AzureMonitorTraceExporter} that
- * implements {@link SpanExporter} interface defined by OpenTelemetry API specification.
+ * This class provides a fluent builder API to configure the OpenTelemetry SDK with Azure Monitor Exporters.
  */
 public final class AzureMonitorExporterBuilder {
 
-    private static final ClientLogger LOGGER = new ClientLogger(AzureMonitorExporterBuilder.class);
-
-    private static final String APPLICATIONINSIGHTS_CONNECTION_STRING =
-        "APPLICATIONINSIGHTS_CONNECTION_STRING";
-    private static final String APPLICATIONINSIGHTS_AUTHENTICATION_SCOPE =
-        "https://monitor.azure.com//.default";
-
-    private static final Map<String, String> PROPERTIES =
-        CoreUtils.getProperties("azure-monitor-opentelemetry-exporter.properties");
-
     private ConnectionString connectionString;
-    private TokenCredential credential;
-
-    // suppress warnings is needed in ApplicationInsights-Java repo, can be removed when upstreaming
-    @SuppressWarnings({"UnusedVariable", "FieldCanBeLocal"})
-    private AzureMonitorExporterServiceVersion serviceVersion;
 
-    private HttpPipeline httpPipeline;
+    // http pipeline configuration
+    private TokenCredential credential;
     private HttpClient httpClient;
     private HttpLogOptions httpLogOptions;
-    private RetryPolicy retryPolicy;
     private final List<HttpPipelinePolicy> httpPipelinePolicies = new ArrayList<>();
-
-    private Configuration configuration = Configuration.getGlobalConfiguration();
```

Comment:
I removed Configuration since we are leveraging OpenTelemetry Java's `ConfigProperties` and so exposing two different configuration sources could be confusing